assets/section-featured-product.css

```diff
@@ -49,15 +54,15 @@
 }
 
 @media screen and (min-width: 990px) {
-  .background-secondary .featured-product:not(.product--no-media) > .product__info-wrapper {
+  .background-secondary .featured-product > .product__info-wrapper {
     padding: 0 0 0 5rem;
   }
 
-  .background-secondary .featured-product:not(.product--no-media).product--right > .product__info-wrapper {
+  .background-secondary .featured-product.product--right > .product__info-wrapper {
     padding: 0 5rem 0 0;
   }
 
-  .featured-product:not(.product--no-media) > .product__info-wrapper {
+  .featured-product > .product__info-wrapper {
```

Comment:
I'm not sure this needs this tweaking as it might break things for no media products.